addons/mail/wizard/mail_wizard_invite.py

```diff
@@ -42,7 +43,9 @@ def default_get(self, fields):
     partner_ids = fields.Many2many('res.partner', string='Recipients', help="List of partners that will be added as follower of the current document.",
                                    domain=[('type', '!=', 'private')])
     message = fields.Html('Message')
-    send_mail = fields.Boolean('Send Email', default=True, help="If checked, the partners will receive an email warning they have been added in the document's followers.")
+    notify = fields.Boolean('Send Notification', default=True,
```

Comment:
Don't forget the migration script to rename the field :) (target 15.5, if the freeze happens we will redirect to 16.1)

Reply:
As you mentioned this morning, it is still ok to target 15.5, is it ? (commit already targeting  15.5 if I’m not mistaken)

Reply:
We will target master (16.1), as this is not really crucial for the upcoming v16.